serve/mlc_serve/engine/sampling_params.py

```diff
@@ -46,13 +45,17 @@ class SamplingParams:
             to -1 to consider all tokens.
         logit_bias: The bias applied on the logit before sampling. Must be in
             [-100, 100].
+        logit_bias_index: Internal data container for indices extracted from logit_bias.
```

Comment:
This comment is not clear, its an "index" to what "Internal data container"

Reply:
It basically manages indices and values of dictionary `logit_bias`. To make it clear, I changed this to 
```
        logit_bias_index: Internal data container that stores indices of `logit_bias`.
        logit_bias_value: Internal data container that stores values of `logit_bias`.
```